<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>53</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Doors</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Temperature Max</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_size">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Small</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Large</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extra Large</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinner_door">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Temperature Min</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Features</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_features">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Tablet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ice Dispenser</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Water Dispenser</string>
     </property>
    </item>
   </widget>
   <widget class="QListView" name="facts_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QListView" name="reco_results">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(19, 26, 63);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_fact">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>20</width>
      <height>361</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="search_facts">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_facts">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Input Facts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Search Results</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Recommendations</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_to_cart">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);
color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Add to cart</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Cart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="purchase">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);
color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Purchase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_from_cart">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);
color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_cart">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);
color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_accessories">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>door shelf</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>water filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>egg tray</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ice maker</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>water valve</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fridge drawer</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);</string>
    </property>
    <property name="text">
     <string>Accessories</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_size">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="add_doors">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="add_max_temp">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="add_min_temp">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="add_feature">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="add_accessory">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(215, 241, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>button_icons/add.png</normaloff>button_icons/add.png</iconset>
    </property>
   </widget>
   <widget class="QTableView" name="shop_results">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(19, 26, 63);</string>
    </property>
   </widget>
   <widget class="QTableView" name="cart">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>361</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(19, 26, 63);</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="max_temp_spinner">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="minimum">
     <double>-20.000000000000000</double>
    </property>
    <property name="maximum">
     <double>15.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="min_temp_spinner">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(19, 26, 63);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="decimals">
     <number>2</number>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>4.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>-18.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumulti_agents">
    <property name="title">
     <string>multi agents</string>
    </property>
   </widget>
   <addaction name="menumulti_agents"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
